suffuse/util.go

```diff
@@ -97,3 +97,9 @@ func ScratchFile() Path {
   deleteOnExit = append(deleteOnExit, path)
   return path
 }
+
+// https://github.com/golang/go/issues/1312
+func FileExists(path string) bool {
+  _, err := os.Stat(path)
```

Comment:
In a decent language like scala one could use any number of mechanisms to write code like
```scala
followingLinks { x.Exists() }
```
Obviously that's completely hopeless here.